src/Pages/Applications/DevtronApps/Details/AppConfigurations/MainContent/types.ts

```diff
@@ -214,22 +219,37 @@ export type NoOverrideEmptyStateProps = {
     environmentName: string
     handleCreateOverride: () => void
     handleViewInheritedConfig: () => void
+    hideOverrideButton?: boolean
 } & (NoOverrideEmptyStateCMCSProps | NoOverrideEmptyStateDeploymentTemplateProps)
 
-type DeploymentTemplateDiffViewConfigType =
+type CompareConfigDiffViewConfigType =
```

Comment:
We should have separate type of DT and CM/CS like currentEditorConfig: DTCompareViewConfigType | CMCSCompareViewConfigType